client/signup/jetpack-connect/index.jsx

```diff
@@ -17,7 +17,7 @@ import Main from 'components/main';
 import JetpackConnectNotices from './jetpack-connect-notices';
 import SiteURLInput from './site-url-input';
 import { getSiteByUrl } from 'state/sites/selectors';
-import { requestSites } from 'state/sites/actions';
+import QuerySites from 'components/data/query-sites';
```

Comment:
Keep in mind QuerySites is not really usable until we land https://github.com/Automattic/wp-calypso/pull/6477